src/components/Comments.jsx

```diff
@@ -53,6 +51,34 @@ const Comments = ({ review }) => {
     }
   };
 
+  const handleDeleteComment = (comment_id) => {
```

Comment:
Could you include some feedback message to indicate if message delete was successful / unsuccessful?